doc/connectivity/region_extraction.rst

```diff
@@ -34,50 +34,50 @@ which is already preprocessed and publicly available at
 datasets.
 
 
-.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_dictlearn_maps.py
     :start-after: # utilities
     :end-before: ################################################################################
 
 .. currentmodule:: nilearn.decomposition
 
-Data decomposition using Canonical ICA
-======================================
+Brain maps using Dictionary Learning
```

Comment:
Dictionary Learning is a modern matrix decomposition technique, broadly related to ICA and PCA. It assumes that neural activity data result from recombinations of a given number of latent (sparse) whole-brain patterns to be discovered.